persistence/persistence.py

```diff
@@ -0,0 +1,24 @@
+#! python
+
+
+def additive_persistence(n):
+    final_count = 0
+    while True:
+        if len(str(n)) == 1:
+            return final_count
+        else:
+            n = sum(map(int, str(n)[:]))
+            final_count += 1
+
+
+def multiplicative_persistence(n):
+    final_count = 0
+    while True:
+        if len(str(n)) == 1:
+            return final_count
+        else:
+            _total = 1
+            for value in map(int, str(n)[:]):
```

Comment:
Maybe you could take `map(int, str(n)[:]` and convert it to its own `def` and reuse it in both of these functions?
Something like:
```
def mapper(value: int) -> map:
    """Create an int map from an incoming value."""
    return map(int, str(value)[:])
```
and then use it as:
`for value in mapper(n):`
and 
`sum(mapper(n))` above?

Reply:
One step ahead of you - I turned it into a lambda ;)